docs/source/index.rst

```diff
@@ -2,12 +2,17 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+.. image:: images/MAMElogo.svg
+    :width: 800px
```

Comment:
Does this need a fixed width?  can you not tell it to use 100 percent width?

Reply:
The docs suggest "400-800px" specifically. I'll try it with 100% and we'll see how it looks.